pokedex/data/csv/locations.csv

```diff
@@ -672,3 +672,4 @@ id,region_id,identifier
 688,6,kiloude-station
 689,6,ambrette-aquarium
 690,6,unknown-dungeon
+691,3,battle-resort
```

Comment:
This needs corresponding entries in location_names.

tbh i'm surprised the battle resort isn't already present — maybe we never did a full load of OR/AS locations?